build/stf-run-ci/tasks/pre-clean.yml

```diff
@@ -48,6 +48,47 @@
         displayName: OperatorHub.io Operators
         publisher: OperatorHub.io
 
+- name: Remove CloudOps CatalogSource if it is installed
+  k8s:
+    state: absent
+    definition:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: CatalogSource
+      metadata:
+        name: service-telemetry-framework-operators
+        namespace: "{{ namespace }}"
+      spec:
+        displayName: CloudOps Operators
+        publisher: CloudOps
+        sourceType: grpc
+
+- name: Remove Service Telemetry Operator bundle build
+  k8s:
+    state: absent
+    api_version: build.openshift.io/v1
+    kind: Build
+    namespace: "{{ namespace }}"
+    label_selectors:
```

Comment:
I never did get label_selectors to work in a k8s lookup... only in k8s_info... not sure if a bug, or just how I was using it.